modules/ROOT/pages/installation/system_requirements.adoc

```diff
@@ -16,42 +16,42 @@ Server
 
 [cols="1,2a",options="header"]
 |===
-|Platform 
+|Platform
 |Options
 
-|Operating System 
-| 
+|Operating System
+|
 * Ubuntu 16.04 and 18.04
-* Debian 7, 8 and 9; 
-* SUSE Linux Enterprise Server 12 with SP1, SP2 and SP3 
-* Red Hat Enterprise Linux/Centos 6.9, 7.3, 7.4, and 7.5 
-* Fedora 26, 27 and 28 
+* Debian 7, 8 and 9;
+* SUSE Linux Enterprise Server 12 with SP1, SP2 and SP3
+* Red Hat Enterprise Linux/Centos 6.9, 7.3, 7.4, and 7.5
+* Fedora 26, 27 and 28
 * openSUSE Tumbleweed and Leap 42.1, 42.2, 42.3
 
-|Database 
+|Database
 |
 * MySQL or MariaDB 5.5+
 * Oracle 11g
 * PostgreSQL
 * SQLite
 
-|Web server 
+|Web server
 |* Apache 2.4 with `prefork` apache-mpm-label and `mod_php`
 
-|PHP Runtime 
+|PHP Runtime
 |* 5.6+, 7.0, & 7.1
 |===
 
-[NOTE] 
+[NOTE]
 ====
-ownCloud 10.1 requires a minimum php version of 7.1. 
+ownCloud 10.1 requires a minimum php version of 7.1.
```

Comment:
Is this correct? Is the support for earlier versions dropped in 10.1? Totally awesome if so.

Reply:
Well, when 10.1 comes out we have to decide ;)

Reply:
core 10.1 or 11.0 or whatever number it is given is highly unlikely to happen before Christmas.
Death of PHP 5.6 and 7.0 security support is a 2018 Christmas gift anyway.
Given the above, the logical conclusion is PHP 7.1 minimum.